### ISSUE TITLE
Exact Curator Count

### ISSUE THREAD

**Issue by DogCatPuppyLover** [ScratchAddons/ScratchAddons#2155]
**Is your feature request related to a problem? Please describe.**

When counting curators in a studio, it's tedious to have to load all the curators in a studio manually.

**Describe the solution you'd like**

My intent is that the Show Exact Count addon would include curator counts. I see multiple ways this could be accomplished; one is by loading each page, similar to how the exact project count works. Another would be to use the API to access the curator count. The latter might not be a viable solution, but if you go to the network tab for the [studios page in My Stuff](https://scratch.mit.edu/mystuff/#galleries), you'll notice that it requests [this](https://scratch.mit.edu/site-api/galleries/all/) page from the API. This contains "projecters_count" (interesting spelling choice, lol) which I believe is the number of projects. You can also find other pages for other "sort by" options. Again, I don't know if the latter option would be viable, but it's worth considering.

**Describe alternatives you've considered**

I coded a little script to count curators on the curator page, once you've loaded all of them: 

`alert(document.getElementById("tabs-content").getElementsByTagName("li").length);`

However, it would be much easier to just programmatically load the pages and count them.

I actually started writing a userscript (not an addon) to do this task of counting curators automatically; however, I haven't continued working on that.

Of course, you can also count them manually. However, that would be a lot of work in large studios.

**Comment by WorldLanguages** [comment]
> This contains "projecters_count" (interesting spelling choice, lol) which I believe is the number of projects.

IIRC it's capped at 101

Also keep in mind, having to load every curator page every time a user with the addon visits a studio would be pretty spammy to Scratch servers

**Comment by DogCatPuppyLover** [comment]
> > This contains "projecters_count" (interesting spelling choice, lol) which I believe is the number of projects.
> 
> IIRC it's capped at 101
> 
> Also keep in mind, having to load every curator page every time a user with the addon visits a studio would be pretty spammy to Scratch servers

Doesn't counting projects have the same problem?

**Comment by DogCatPuppyLover** [comment]
The `curators_count`, however, is not capped at 101.

**Comment by CubeyTheCube** [comment]
> > > This contains "projecters_count" (interesting spelling choice, lol) which I believe is the number of projects.
> > 
> > 
> > IIRC it's capped at 101
> > Also keep in mind, having to load every curator page every time a user with the addon visits a studio would be pretty spammy to Scratch servers
> 
> Doesn't counting projects have the same problem?

I'm pretty sure the server has data stored for each user for that, so there's no need to count the projects; it just isn't part of Scratch's API.

**Comment by DogCatPuppyLover** [comment]
> > > > This contains "projecters_count" (interesting spelling choice, lol) which I believe is the number of projects.
> > > 
> > > 
> > > IIRC it's capped at 101
> > > Also keep in mind, having to load every curator page every time a user with the addon visits a studio would be pretty spammy to Scratch servers
> > 
> > 
> > Doesn't counting projects have the same problem?
> 
> I'm pretty sure the server has data stored for each user for that, so there's no need to count the projects; it just isn't part of Scratch's API.

![Screenshot 2021-04-14 145141](https://user-images.githubusercontent.com/79656307/114785050-f50a0e00-9d30-11eb-9605-b59e40262e71.png)

The full URL for those last requests is https://api.scratch.mit.edu/studios/5842709/projects/?limit=40&offset=[offset]

I'm pretty sure that that's SA counting projects; when I disable SA, those don't appear.

**Comment by WorldLanguages** [comment]
@DogCatPuppyLover yes but that's the JSON API which is way more robust, not sure if there's an endpoint for curators yet

**Comment by GrahamSH-LLK** [comment]
> @DogCatPuppyLover yes but that's the JSON API which is way more robust, not sure if there's an endpoint for curators yet

Nope, it's not open yet.
